src/app/artist-list/artist-list.component.css

```diff
@@ -0,0 +1,82 @@
+input[type="text"] {
+  width: 130px;
+  box-sizing: border-box;
+  border: 2px solid #ccc;
+  border-radius: 4px;
+  font-size: 16px;
+  background-color: white;
+  background-image: url("~/src/assets/search--v2.png");
```

Comment:
The image being reference here does not exist.

Reply:
@strizzwald fixed